### DISCUSSION TITLE
strobing light/dark mode error

### DISCUSSION THREAD

**Discussion by dmvjs** [chakra-ui/chakra-ui#9048] [General]
### Description

when I go to https://www.chakra-ui.com/ and I click the light/dark mode button in Safari it sometimes immediately begins strobing light and dark mode, and this happens for all pages. I was trying to implement the light/dark mode feature from chakra 3 into my page but this is driving me insane.

### Link to Reproduction

https://www.chakra-ui.com/docs/components/concepts/overview

### Steps to reproduce

go to the page
click on the light/dark mode button in Safari
it bugs out

### Chakra UI Version

3

### Browser

Safari 18.1

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by dmvjs** [comment]
https://github.com/user-attachments/assets/e87a70d8-dcbc-4310-8c17-8c0c5b888381

**Comment by dmvjs** [comment]
sometimes it is very very fast, here it is slow ^^^

**Reply by segunadebayo** [reply]
Chakra relies on `next-themes` for dark mode. This means it's likely an issue with `next-themes` in Safari.

Let's consider opening a discussion there as well.
